agent/__init__.py

```diff
@@ -367,7 +367,7 @@ def get_uptime():
 
 
 def get_open_ports(dev=False):
-    connections, ports = security_helper.netstat_scan()
+    _, ports = security_helper.netstat_scan()
     return ports
```

Comment:
`return security_helper.netstat_scan()[0]`

Reply:
It's `[1]`. See, you've already made a mistake yourself. I believe we should specify what exactly we want to return.

Reply:
Yes, I used a wrong index but the solution is quite correct. I don't insist on using it if you prefer to introduce a couple of useless variables for the sake of readability instead of adding a short comment.